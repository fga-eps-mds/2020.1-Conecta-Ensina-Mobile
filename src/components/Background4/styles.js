import styled from 'styled-components/native';

export const BlueContainer = styled.View`
  background-color: ${(props) => props.theme.colors.fundoAzul};
  flex: 1;
  display: flex;
  justify-content: flex-end;
`;

export const WhiteContainer = styled.View`
  background-color: ${(props) => props.theme.colors.branco};
  flex: 78;
  display: flex;
  justify-content: center;
  border-top-left-radius: 20px;
  border-top-right-radius: 20px;
`;

export const UserContainer = styled.View`
  flex-direction: row-reverse;
  flex: 10;
  margin-left: 30px;
  align-items: center;
`;
