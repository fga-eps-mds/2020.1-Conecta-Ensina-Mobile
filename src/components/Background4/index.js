import React from 'react';
import Theme from '../../../Theme';
import {BlueContainer, WhiteContainer, UserContainer} from './styles';

export default function Background1({children}) {
  return (
    <Theme>
      <BlueContainer>
        <UserContainer />
        <WhiteContainer>{children}</WhiteContainer>
      </BlueContainer>
    </Theme>
  );
}
