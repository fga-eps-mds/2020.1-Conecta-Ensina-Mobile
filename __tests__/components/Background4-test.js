import React from 'react';
import 'react-native';
import {render} from '@testing-library/react-native';
import Background4 from '../../src/components/Background4';

describe('Testing Background 4', () => {
  test('should renders component', () => {
    const tree = render(<Background4 />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
