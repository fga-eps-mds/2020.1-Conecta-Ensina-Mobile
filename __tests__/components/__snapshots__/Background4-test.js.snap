// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Background 4 should renders component 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "#184E79",
        "display": "flex",
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
        "justifyContent": "flex-end",
      },
    ]
  }
>
  <View
    style={
      Array [
        Object {
          "alignItems": "center",
          "flexBasis": 0,
          "flexDirection": "row-reverse",
          "flexGrow": 10,
          "flexShrink": 1,
          "marginLeft": 30,
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "backgroundColor": "#ffffff",
          "borderTopLeftRadius": 20,
          "borderTopRightRadius": 20,
          "display": "flex",
          "flexBasis": 0,
          "flexGrow": 78,
          "flexShrink": 1,
          "justifyContent": "center",
        },
      ]
    }
  />
</View>
`;
